pallets/ds-accounts/src/tests.rs

```diff
@@ -0,0 +1,265 @@
+use crate::mock::*;
+use frame_support::{
+    assert_noop, assert_ok,
+};
+
+// Learn more about testing substrate runtime modules
+// https://substrate.dev/docs/en/knowledgebase/runtime/tests
+// type Module = super::Module<Test>;
+type Timestamp = pallet_timestamp::Module<Test>;
+type System = frame_system::Module<Test>;
+type Balances = pallet_balances::Module<Test>;
+type Error = super::Error<Test>;
+
+#[allow(dead_code)]
+// get last events and reset
+fn events() -> Vec<TestEvent> {
+    let evt = System::events()
+        .into_iter()
+        .map(|evt| evt.event)
+        .collect::<Vec<_>>();
+    System::reset_events();
+    evt
+}
+
+// Constants to make tests more readable
+const ADMIN_ACCOUNT_ID: u64 = 1;
+const REGISTRAR_1_ACCOUNT_ID: u64 = 2;
+const REGISTRAR_2_ACCOUNT_ID: u64 = 3;
+const PILOT_1_ACCOUNT_ID: u64 = 4;
+
+#[allow(dead_code)]
+fn last_event() -> TestEvent {
+    System::events().pop().expect("Event expected").event
+}
+
+#[test]
+fn it_default_pallet_transaction_payment_multiplier() {
+    use frame_support::sp_runtime::{traits::Convert, FixedPointNumber, FixedU128};
+
+    type Multiplier = FixedU128;
+    let multiplier: Multiplier = Multiplier::saturating_from_integer(1);
+
+    let new_multiplier = <() as Convert<Multiplier, Multiplier>>::convert(multiplier);
+    println!(
+        "default multiplier convert from {:} to {:}",
+        multiplier, new_multiplier
+    );
+
+    assert_eq!(new_multiplier, 0.into());
+}
+
+#[test]
+#[cfg(feature = "payment")]
+fn it_identity_pallet_transaction_payment_multiplier() {
+    use frame_support::sp_runtime::{traits::Convert, FixedPointNumber, FixedU128};
+
+    type Multiplier = FixedU128;
+    let multiplier: Multiplier = Multiplier::saturating_from_integer(1);
+
+    let new_multiplier = <crate::payment::IdentityMultiplierUpdater as Convert<
+        Multiplier,
+        Multiplier,
+    >>::convert(multiplier);
+    println!(
+        "identiry multiplier convert from {:} to {:}",
+        multiplier, new_multiplier
+    );
+
+    assert_eq!(new_multiplier, multiplier);
```

Comment:
Generally it is a bad practice to have `println!`s in tests.

Better approach is to write proper asserts with extended explanation of what's happened. In case of a successful test it will not pollute stdout with debug ouptut, and in case of a failure it will print exactly what is needed.

For example:
```
assert_eq!(
    new_multiplier,
    multiplier
    "identiry multiplier convert FAILED from {:} to {:}",
    multiplier, new_multiplier
);
```

or something like that.

Reply:
BTW it is a typo in word "identity" here :)